lib/visitor-keys.js

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable jsdoc/valid-types -- doesn't allow `readonly`. TODO: check why */
```

Comment:
This is also report in https://github.com/jsdoc-type-pratt-parser/jsdoc-type-pratt-parser/issues/164